config/jobs/cert-manager/release-previous/cert-manager-release-previous-periodics.yaml

```diff
@@ -479,3 +479,61 @@ periodics:
       options:
       - name: ndots
         value: "1"
+
+- name: ci-cert-manager-previous-e2e-v1-18
```

Comment:
Does this mean we hadn't been periodically running e2e tests for CM 0.15 on k8s 1.18?
Was that just an oversight? Or is there a reason that 0.15 wasn't tested on k8s 1.18?

Reply:
we did not as on time of release k8s was not released, we did always test master with it